### PR TITLE
docs(reuse): updates of weekly meeting

### PR DESCRIPTION
**Pull request by rohitpandey49**
Signed-off-by: Rohit Pandey <rohit.pandey4900@gmail.com>

## Project

REUSE

### Changes

Meeting update added.

### CODE REVIEW

File: docs/2022/reuse/updates/2022-06-02.md [hunk 1]
```diff
@@ -103,3 +103,86 @@ GSoC 2022 Discussions - REUSE
...
+*(June 16th, 2022)*
+
+GSoC 2022 weekly update
+
+### Attendees
+
+- [Shaheem Azmal M MD](https://github.com/shaheemazmalmmd)
+- [Vivek Kumar](https://github.com/viv9k)
+- [Avinal Kumar](https://github.com/avinal)
+- [Sahil Jha]( https://github.com/sjha2048)
```
Comment (GMishx) [inline docs/2022/reuse/updates/2022-06-02.md:177]:
```suggestion
- [Sahil Jha](https://github.com/sjha2048)
```

File: docs/2022/reuse/updates/2022-06-02.md [hunk 2]
```diff
@@ -103,3 +103,86 @@ GSoC 2022 Discussions - REUSE
...
+- [Sushant Kumar](https://github.com/its-sushant)
+- [Rohit Pandey](https://github.com/rohitpandey49)
+- [Feng Wenhan]( https://github.com/fwhdzh)
```
Comment (GMishx) [inline docs/2022/reuse/updates/2022-06-02.md:180]:
```suggestion
- [Feng Wenhan](https://github.com/fwhdzh)
```

File: docs/2022/reuse/updates/2022-06-02.md [hunk 3]
```diff
@@ -103,3 +103,86 @@ GSoC 2022 Discussions - REUSE
...
+- [Soham Banerjee](https://github.com/soham4abc)
+
+### Discussions
+
+- Explored the study material/blog links shared by mentors. 
+- **Can I raise a pull request with a few changes for review?**
+    - Yes. You can raise PR.
+
+## Meeting 4
+*(June 13th, 2022)*
```
Comment (GMishx) [inline docs/2022/reuse/updates/2022-06-02.md:135]: Hello @rohitpandey49 ,
As from June 13, the coding period started, so please move (this and following) updates to new weekly files under same folder as done in previous run:
https://github.com/fossology/gsoc/tree/main/docs/2021/copyrights/updates

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/reuse/updates/2022-06-02.md (hunk 3 above)
Comment (rohitpandey49) [reply docs/2022/reuse/updates/2022-06-02.md]: Done.

**Pull request merged**